aeon/anomaly_detection/tests/test_lof.py

```diff
@@ -0,0 +1,250 @@
+"""Tests for the LOF class."""
+
+import numpy as np
+import pytest
+
+from aeon.anomaly_detection import LOF
+from aeon.testing.data_generation import make_example_1d_numpy
+from aeon.utils.validation._dependencies import _check_soft_dependencies
+
+
+@pytest.mark.skipif(
+    not _check_soft_dependencies("pyod", severity="none"),
+    reason="required soft dependency PyOD not available",
+)
+def test_lof_default():
+    """Test LOF with default parameters."""
+    series = make_example_1d_numpy(n_timepoints=80, random_state=0)
+    series[50:58] -= 2  # Introducing anomalies
+
+    # Define common parameters
+    n_neighbors = 20
+    algorithm = "auto"
+    leaf_size = 30
+    metric = "minkowski"
+    p = 2
+    novelty = False
+    n_jobs = 1
+    window_size = 1  # Set window_size to 1 for point-based processing
+    stride = 1  # Set stride to 1
+
+    # Initialize aeon LOF with window_size=1 and stride=1
+    lof_aeon = LOF(
+        n_neighbors=n_neighbors,
+        algorithm=algorithm,
+        leaf_size=leaf_size,
+        metric=metric,
+        p=p,
+        n_jobs=n_jobs,
+        window_size=window_size,
+        stride=stride,
+    )
+    scores_aeon = lof_aeon._fit_predict(series)
```

Comment:
Always use the original methods!
```suggestion
    scores_aeon = lof_aeon.fit_predict(series)
```